components/dropdown/demo/dropdown-button.html

```diff
@@ -10,6 +10,7 @@
 	<script type="module">

```

Comment:
Now that this contains all the openers, I'd probably rename it to `dropdown-openers.html`. Presumably the other openers (context-menu (chevron icon) and more (... icon) will be added here once converted?